frontend/admin/src/js/components/user/util.ts

```diff
@@ -0,0 +1,103 @@
+import {
+  JobLookingStatus,
+  LanguageAbility,
+  OperationalRequirement,
+  WorkRegion,
+} from "../../api/generated";
+
+// convert string type to Enum types for various selections
+export function stringToEnumLanguage(
+  selection: string,
+): LanguageAbility | undefined {
+  if (selection === LanguageAbility.Bilingual) {
+    return LanguageAbility.Bilingual;
+  }
+  if (selection === LanguageAbility.English) {
+    return LanguageAbility.English;
+  }
+  if (selection === LanguageAbility.French) {
+    return LanguageAbility.French;
+  }
```

Comment:
This should be the same without explicitly mentioning each:

```suggestion
  if (Object.values(LanguageAbility).includes(selection as LanguageAbility)) {
    return selection as LanguageAbility;
  }
```

Reply:
There maybe a more general way to do this in typescript (so one function works for all of these), but I'm unsure what this is off the top of my head :)